app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  before_save{self.email = email.downcase}
+  validates :name, presence: true, length: {maximum: 50}
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  validates :email, presence: true, length: {maximum: 255},
+            format: {with: VALID_EMAIL_REGEX},
+            uniqueness: true
+  has_secure_password
+  validates :password, presence: true, length: {minimum: 6}
```

Comment:
```suggestion
  validates :password, presence: true, length: {minimum: 6}
  
  has_secure_password
```
Sao chia cắt con ng ta vậy, nó buồn đó

ngoài ra, cá giá trị cố định như 50 6 255 em nên khai báo vào setting file nhé
tìm hiểu gem config để tạo dc file settinsg nhé bé